service/src/main/java/uk/gov/hmcts/reform/fpl/enums/UserRole.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public enum UserRole {
+    DEFAULT {
```

Comment:
This role shouldn't be accessible in the code.

Reply:
now updated.